doc/tutorials/chapter_6/00_combinational_logic.md

```diff
@@ -0,0 +1,383 @@
+# Content
+
+- [What is Combinational Logic?](#what-is-combinational-logic)
+- [What is Conditionals?](#what-is-conditionals)
+- [If, ElseIf, Else](#if-elseif-else)
+  - [Start by declaring a conditional Block](#start-by-declaring-a-conditional-block)
+  - [Add the condition inside the conditional block](#add-the-condition-inside-the-conditional-block)
+- [Case](#case)
+  - [Start by declaring a case](#start-by-declaring-a-case)
+  - [Add Expressions](#add-expressions)
+  - [Add Case Items](#add-case-items)
+  - [Add Default Items](#add-default-items)
+  - [Encapsulate case into a Combinational](#encapsulate-case-into-a-combinational)
+- [Exercises](#exercises)
+
+## Learning Outcome
+
+In this chapter:
+
+- You will learn how to create a combinational logic that are equivalent to System Verilog `always_comb`. We will review on what is conditionals and how to use combinational condition function such as `If..Else` and `Case`.
+
+## What is Combinational Logic?
+
+There are two types of digital circuit, which are combinational logic and sequential logic. As for this chapter, we will look into combinational logic circuit.
+
+A combinational circuit's outputs depends only on the current values of the inputs; in other words, it combines just the current input values to compute the output. For example, a logic gate is a combinational circuit.A circuit is combinational if it consists of interconnected circuit elements such that:
+
+- Every circuit element is itself combinational.
+- Every node of the circuits is either designated as an input to the circuit or connects to exactly one output terminal of a circuit element.
```

Comment:
What does this bullet mean?  I don't think this is a restriction on the definition of a conditional circuit?

Reply:
This bullet still doesn't make sense to me.  Can you reword or explain what you're trying to say?

Reply:
I am trying to say what is the criteria to determine if the circuit itself is combinational. Not sure, should I mention that or just omit this part?

Reply:
I now removed the bullet points. I wonder if the sentence is clearer now.

Reply:
This doesn't make sense to me:
> nodes being inputs or connecting to one output terminal

Combinational logic just means it has no sequential elements, i.e. no state.  It has nothing to do with a "node being an input" or "connecting to an output".  You can have long chains gates where the gates in the middle are not connected to an "input" or "output" of any larger structure, but are still combinational in nature.

This is also a little confusing:
> synchronous operation

Yes, it's synchronous, but usually people talk about synchronous vs asynchronous in the context of sequential logic.

Reply:
Let me remove these two from the paragraph.